content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
+---
+
+{{< callout url=""https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
+Risk Based Entity Insights is in private beta.
+{{< /callout >}}
+
+## Overview
+
+[Cloud SIEM's Risk Based Entity Insights][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
+
+With Risk Based Entity Insights, you can:
+
+- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
+- Triage relevant items in bulk.
+- Take mitigation steps such as creating an automated workflow or creating a global case for an entity.
+
+## Explore Entities
+
+### Query and filter entities
+
+On the [Entities Explorer page][4], you can view all entities that have at least one signal.
+
+{{< img src="security/entities/entities-explorer.png" alt="A list of entities and their risk scores in the Entities Explorer" style="width:100%;" >}}
+
+### Quickly build context on an entity
+
+Click an entity in the [Explorer][4] to open the entity details panel.
+
+{{< img src="security/entities/entity-side-panel.png" alt="The details panel for an entity" style="width:90%;" >}}
+
+The top of the panel displays the count of related signals, identity risks, and so on, with links to the list view of each.
+
+The **Entity Context** section displays entity's attributes, such as the entity type and the list of fired signals. 
```

Comment:
The **What contributes** section displays the list of fired signals, relevant misconfigurations, and identity risks.